op-service/log/cli.go

```diff
@@ -53,7 +53,7 @@ func CLIFlagsWithCategory(envPrefix string, category string) []cli.Flag {
 		},
 		&cli.GenericFlag{
 			Name:     FormatFlagName,
-			Usage:    "Format the log output. Supported formats: 'text', 'terminal', 'logfmt', 'json', 'json-pretty',",
+			Usage:    fmt.Sprintf("Format the log output. Supported formats: %s", SupportedFormatsString()),
```

Comment:
This would cause the enum to be 

```
text, terminal, logfmt, logfmtms, json, jsonms
```

So some new values (logfmtms, jsonms) added and some old values removed (json-pretty).

Is that ok?

Reply:
better than ok, I think json-pretty was unsupported actually. If you look at the bottom of this file you'll see:

```go
// FormatHandler returns the correct slog handler factory for the provided format.
func FormatHandler(ft FormatType, color bool) func(io.Writer) slog.Handler {
```

and this function does not handle `json-pretty`, nor is there any mention of the this format type anywhere in the rest of the codebase.

Reply:
Sweet, that makes sense.